core/src/wsv.rs

```diff
@@ -183,31 +190,81 @@ impl<W: WorldTrait> WorldStateView<W> {
     /// you likely have data corruption.
     #[iroha_futures::telemetry_future]
     #[log(skip(self, block))]
+    #[allow(clippy::expect_used)]
     pub async fn apply(&self, block: VersionedCommittedBlock) -> Result<()> {
-        // TODO: Validate the trigger executables as well as the technical account.
-        for Action {
-            technical_account,
-            executable,
-            ..
-        } in &block.as_v1().trigger_recommendations
-        {
-            self.process_executable(executable, technical_account)?;
-            task::yield_now().await;
-        }
+        let time_event = Event::Time(TimeEvent(TimeInterval::new(
+            Duration::from_millis(block.as_v1().header.timestamp.try_into()?),
+            Duration::from_millis(self.config.commit_time_ms),
```

Comment:
Is having `commit_time_ms` appropriate? I guess the current implementation does not cover the time axis

Reply:
Answered in the comment above